src/translators/sql_translator.py

```diff
@@ -1105,3 +1197,68 @@ def _get_entity_type(self, entity_id, fiware_service):
 
     def _compute_type(self, attr_t, attr):
         raise NotImplementedError
+
+    def _execute_query_via_cache(self, table_name, key, stmt, parameters=None,
+                                 ex=None):
```

Comment:
this function provide a way to check if a query is in cache in a certain key, table tuple. if not execute the query.